packages/dynamic-import/server.js

```diff
@@ -66,7 +66,11 @@ function middleware(request, response) {
   response.setHeader("Access-Control-Allow-Origin", "*");
 
   if (request.method === "OPTIONS") {
-    response.setHeader("Access-Control-Allow-Headers", "*");
+    if (request.headers["access-control-request-headers"] !== undefined) {
```

Comment:
Do we need to worry about different possible capitalizations when checking for this header?

Also we typically test `typeof value === "undefined"` rather than relying on the `undefined` identifier, just because `undefined` is mutable in some environments.